File: Programmers/c++/가운데 글자 가져오기.cpp
/*
프로그래머스 가운데 글자 가져오기
홀수면 s.length/2
짝수면 s.length/2-1 s.length/2
글자 가져오기
*/
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
	string answer = "";
	int len = s.length();
	if (len % 2 == 1) {
		answer.push_back(s[len / 2]);
	}
	else {
		answer.push_back(s[len / 2 - 1]);
		answer.push_back(s[len / 2]);
	}
	return answer;
}

File: Programmers/javascript/가운데 글자 가져오기.js
/*
프로그래머스 가운데 글자 가져오기
홀수면 s.length/2->정수변환
짝수면 s.length/2-1 s.length/2 
글자 가져오기
*/
function solution(s) {
    var answer = '';
    if(s.length%2==0){
        answer=s[s.length/2-1]+s[s.length/2];
    }else{
        answer=s[parseInt(s.length/2)];
    }
    return answer;
}
